.github/workflows/modmesh_pytest.yml

```diff
@@ -41,9 +41,9 @@ jobs:
     - name: dependency (macos)
       if: matrix.os == 'macos-latest'
       run: |
-        brew install python3 llvm
-        ln -s "$(brew --prefix llvm)/bin/clang-format" "/usr/local/bin/clang-format"
-        ln -s "$(brew --prefix llvm)/bin/clang-tidy" "/usr/local/bin/clang-tidy"
+        brew install python3 llvm@9
```

Comment:
This doesn't seem to address the root cause.  The newer version `clang-tidy` (llvm 11) performs more checks and then `modmesh_pytest` fails.  Falling back to older version doesn't address the additional violation detected.

If the new violation is a true issue, it should get fixed.  If it's a false alarm or a fix is impractical, it should be explicitly excluded from the check.  Using older version is not the right approach to either of the treatments.

Reply:
r means review.
alright, then we should fix the code for llvm11.

Reply:
If it's easy to fix, we should fix it.  If you think it's a false alarm, let's discuss how to suppress it.

Reply:
Yes, I think all of them are false alarms. I will postpone the fix to maybe next week since it takes me too much time today...